spec/mailers/user_mailer_spec.rb

```diff
@@ -3,6 +3,17 @@
 require 'rails_helper'
 
 RSpec.describe UserMailer do
+  shared_examples 'delivery to memorialized user' do
+    context 'when user is not active for authentication' do
```

Comment:
I would change the description, as “not active for authentication” sounds more like `locked` than `memorial`.
```suggestion
    context 'when the account is memorialized' do
```